cli/connhelper/connhelper.go

```diff
@@ -53,6 +53,16 @@ func GetConnectionHelper(daemonURL string) (*ConnectionHelper, error) {
 	return nil, err
 }
 
+// GetCommandConnectionHelper returns a ConnectionHelp constructed from an arbitrary command.
+func GetCommandConnectionHelper(cmd string, flags ...string) (*ConnectionHelper, error) {
```

Comment:
Wondering why we didn't call the other one `NewConnectionHelper` (and this one `NewCommandConnectionHelper`)

(just thinking out loud 🤔)